java/core/src/java/org/apache/orc/impl/RecordReaderImpl.java

```diff
@@ -318,16 +315,25 @@ public long getNext() {
    * @param <T> the type of the comparision
    * @return the location of the point
    */
-  static <T> Location compareToRange(Comparable<T> point, T min, T max) {
+  static <T> Location compareToRange(Comparable<T> point, T min, T max, boolean isLowerBoundSet, boolean isUpperBoundSet) {
```

Comment:
I suggest that we instead do:
```static <T> Location compareToRange(Comparable<T> point, T min, T max, T lowerBound, T upperBound)```
so the normal case would look like:
```compareToRange(min, max, min, max);```
or in the more general case:
```compareToRange(stats.getMin(), stats.getMax(), stats.getLowerBound(), stat.getUpperBound());```

Reply:
Sure, fixed this.